### ISSUE TITLE
setting columnOrder does not redraw the grid

### ISSUE THREAD

**Issue by domehead100** [TonyGermaneri/canvas-datagrid#160]
Depends on #159 being fixed first in order to be able to set the columnOrder successfully.

### Expected behavior and actual behavior.
I would probably expect that setting the columnOrder property would cause the grid to immediately redraw with the new column order, but it does not unless I call grid.draw() after setting the column order.

Is this by design?  Maybe there's a reason for the current behavior, such as allowing the developer to make multiple changes to multiple properties and then redraw the grid all at once (kind of like a transactional capability)?

### Steps to reproduce the problem.
Having a reference to a grid (e.g., window.grid) execute the following to reverse the column order:
```typescript
window.grid.columnOrder = window.grid.schema.map((_, i) => i).sort((a, b) => b - a);
```
and see that even though the columns have been reversed the grid does not update/redraw.
Then issue 
```typescript
window.grid.draw()
```
and see that the grid now updates.

### Specifications like the version of the project, operating system, or hardware.
Latest version, Windows 10

**Comment by TonyGermaneri** [comment]
Yeah, it seems like it should redraw for you.  Drawing is super cheap.

**Comment by TonyGermaneri** [comment]
For sure let's just add the draw to the setter, if you're able to do it please do open a PR.  Although headless automated testing does not work right now manual testing is a must.  Run `/test/tests.html` on your local machine before you make a PR to make sure all the tests pass.

The reason automated testing is broken is because headless chrome does not draw on a canvas and phantomjs does not support web components, so I'm totally screwed with testing until I have a proper CI/CD environment setup... should make a ticket for that.  I'm using coveralls right now and I'd love to get proper web hook integration and automated tests working again.  It just seems right now that there is no love for complex canvas projects that use free software (not VMs) for testing.

I want to do it, but I am but a single person.  Sometimes the more tickets I add to the project the more daunting it seems work on it.  Thanks for volunteering to help!  Not a lot of that around here.  But I must endure... VQP
